lib/blockscore/candidate.rb

```diff
@@ -19,7 +19,7 @@ def hits
     end
 
     def search(options = {})
-      post "#{api_url}watchlists", options.merge(candidate_id: id)
+      post "#{API_URL}watchlists", options.merge(candidate_id: id)
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/67/files#r52547249.